src/widgets/ProductsList/ProductsList.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react'
```

Comment:
```suggestion
import type { FC } from 'react'
```

FC - это тоже тип, большая молодец, что используешь импорты типов